packages/cli/src/commands/connector/utils.ts

```diff
@@ -86,8 +86,50 @@ export const normalizePackageName = (name: string) =>
     )
     .join('/');
 
+const getConnectorDirectory = (instancePath: string) =>
+  path.join(instancePath, coreDirectory, connectorDirectory);
+
+export const isOfficialConnector = (packageName: string) =>
+  packageName.startsWith('@logto/connector-');
+
+const getConnectorPackageName = async (directory: string) => {
+  const filePath = path.join(directory, 'package.json');
+
+  if (!existsSync(filePath)) {
+    return;
+  }
+
+  const json = await readFile(filePath, 'utf8');
+  const { name } = z.object({ name: z.string() }).parse(JSON.parse(json));
+
+  if (name.startsWith('connector-') || Boolean(name.split('/')[1]?.startsWith('connector-'))) {
```

Comment:
nit: we can extract this name check into a util function, because the name rule may change in the future, the 3rd party connector name may not contain "connector-"

Reply:
great idea. i think we can enforce that for the moment. LOG-4394